corio.py

```diff
@@ -114,6 +119,15 @@ def main(options):
 
     :param options: Parsed Arguments.
     """
+    LOGGER.info("Setting up environment!!")
+    # Check cluster is healthy to start execution.
+
+    if options.degraded_mode:
+        activate_degraded_mode(options)
+        options.number_of_nodes -= literal_eval(os.getenv("DEGRADED_PODS_CNT"))
+
+    if options.health_check:
+        check_health()
```

Comment:
If health check enabled with degraded mode then we need support to skip health check for degraded node.

Reply:
Done, Please check, Skipping health check for degraded nodes.
                if pod_name in ClusterServices.degraded_pods:
                    LOGGER.info("Skipping Check for Pod %s as system is in degraded mode", pod_name)
                    continue

Reply:
Done

Reply:
From code I can see support added if node degraded through tool, what if this option is skipped? Will health check work?

Reply:
Done changes accordingly